packages/ui/src/utils/get-filtered-properties.test.ts

```diff
@@ -0,0 +1,377 @@
+import { KaotoSchemaDefinition } from '../models/kaoto-schema';
+import { getFilteredProperties } from './get-filtered-properties';
+
+describe('getFilteredProperties()', () => {
+  const schemaObject = {
```

Comment:
Could it be possible to move this `schemaObject` to the `stub` folder, perhaps as `RestSchema.ts`?